.github/workflows/check-catalyst.yaml

```diff
@@ -2,6 +2,8 @@ name: Check Catalyst Build
 
 on:
   pull_request:
+    types:
+      - ready_for_review
```

Comment:
I think this means the workflow is no longer triggering on commits right? If I read [the docs](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) correctly, the default values are `opened`, `synchronize`, or `reopened`, which we may want to include here to restore existing behaviour.

Reply:
Ah right yes, will get that updated thanks!